CDCgov/prime-reportstream
# 13825: quiet JWT exception logging

This PR quiets down JWT exception logging. This should solve the issue of constant PagerDuty alerting.

Test Steps:
1. Attempt to authenticate without a valid JWT
2. Check the exceptions table to ensure no `JWTVerificationException` shows up

## Changes
- Move exception out of parameters and add the message directly to the string

### Testing

## Linked Issues
- Fixes #13825
